### PR TITLE
Use rapidsai Conda packages

### PR DESCRIPTION
**Pull request by jakirkham**
Updates the docs to point to the `rapidsai` channel for packages.

### CODE REVIEW

**Comment by jakirkham** [comment]
> Looks good. Can you add a line in the README under the UCX source `git checkout v1.7.x`

This is a good idea. I added it to PR ( https://github.com/rapidsai/ucx-py/pull/351 ) instead just to cutdown on conflicts when merging. Hope that is ok 🙂

File: README.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ with either `9.2`, `10.0`, or `10.1`. Also replace `<processor type>`
 with `cpu` or `gpu`.
 
 ```bash
-conda create -n ucx -c conda-forge -c conda-forge/label/rc_ucx cudatoolkit=<CUDA version> ucx-proc=*=<processor type> ucx ucx-py python=3.7
+conda create -n ucx -c conda-forge -c rapidsai cudatoolkit=<CUDA version> ucx-proc=*=<processor type> ucx ucx-py python=3.7
```
Comment (randerzander) [inline README.md]: Should this be `rapidsai-nightly` for awhile?

We're calling the release alpha, so perhaps we should even include a comment saying that you likely want bugfixes/improvements from the nightly packages until we have a more stable release.

File: docs/source/install.rst [hunk 2]
```diff
@@ -8,16 +8,18 @@ With GPU support:
 
 ::
 
-    conda create -n ucx -c conda-forge -c conda-forge/label/rc_ucx \
+    conda create -n ucx -c conda-forge -c rapidsai \
```
Comment (randerzander) [inline docs/source/install.rst]: Same question about `rapidsai-nightly`

File: docs/source/install.rst [hunk 3]
```diff
@@ -8,16 +8,18 @@ With GPU support:
...
       cudatoolkit=<CUDA version> ucx-proc=*=gpu ucx ucx-py python=3.7
 
 Without GPU support:
 
 ::
 
-    conda create -n ucx -c conda-forge -c conda-forge/label/rc_ucx \
+    conda create -n ucx -c conda-forge -c rapidsai \
```
Comment (randerzander) [inline docs/source/install.rst]: Same question about `rapidsai-nightly`

File: docs/source/quickstart.rst [hunk 4]
```diff
@@ -9,14 +9,14 @@ With GPU support:
 
 ::
 
-    conda create -n ucx -c conda-forge -c conda-forge/label/rc_ucx \
+    conda create -n ucx -c conda-forge -c rapidsai \
```
Comment (randerzander) [inline docs/source/quickstart.rst]: Same question about `rapidsai-nightly`

File: docs/source/quickstart.rst [hunk 5]
```diff
@@ -9,14 +9,14 @@ With GPU support:
...
       cudatoolkit=<CUDA version> ucx-proc=*=gpu ucx ucx-py python=3.7
 
 Without GPU support:
 
 ::
 
-    conda create -n ucx -c conda-forge -c conda-forge/label/rc_ucx \
+    conda create -n ucx -c conda-forge -c rapidsai \
```
Comment (randerzander) [inline docs/source/quickstart.rst]: Same question about `rapidsai-nightly`

**Comment by jakirkham** [comment]
> Not sure if you'd want to do it in this or a follow-on PR, but in trying to follow the build from source section, the instructions say to clone `openucx/ucx` from master, but I believe we actually want branch `v1.7.x`.

I added it to PR ( https://github.com/rapidsai/ucx-py/pull/351 ) instead just to cutdown on conflicts when merging.

> Also shouldn't the conda dependency list should include `libhwloc`?

@quasiben, is handling that in PR ( https://github.com/rapidsai/ucx-py/pull/347 ).

> Should this be rapidsai-nightly for awhile?

Answering here for simplicity. 🙂 No objections. Happy to add that. Though we are also planning to do a release with RAPIDS as part of 0.11. So it won't be long before packages are in `rapidsai` too. WDYT?

**Comment by randerzander** [comment]
> So it won't be long before packages are in `rapidsai` too. WDYT?

Sure, it makes sense to include a note that the release package is available as well.

**Comment by jakirkham** [comment]
I'd be somewhat inclined to make the note say there are also nightly packages as opposed to changing the install commands. Would that work for you?

**Comment by jakirkham** [comment]
@randerzander, is it ok if we tackle this in PR ( https://github.com/rapidsai/ucx-py/pull/351 )? Would like to reduce the number of conflicts we are handling here.

**Comment by jakirkham** [comment]
Thanks all! Let's tackle the remaining issues in PR ( https://github.com/rapidsai/ucx-py/pull/351 ). 😄

**Pull request merged**